app/serializers/user_notification_total_serializer.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+class UserNotificationTotalSerializer < ApplicationSerializer
+  attributes :id,
+             :username,
```

Comment:
Just curious, why do we need to return the user's `id` and `username` to the client? I think the client will already have this information considering this is a logged in required route.

Reply:
We don't need to, it's a convenience. No good use for the user id, I'll remove it, but username seems useful (as a confirmation).